vue-app/src/router/index.ts

```diff
@@ -25,205 +25,213 @@ import VerifyView from '../views/Verify.vue'
-    meta: {
-      title: 'Rounds',
-    },
-  },
-  {
-    path: '/round/:address',
-    name: 'round',
-    component: ProjectList,
-    meta: {
-      title: 'Project List for Round',
-    },
-  },
-  {
-    path: '/about',
-    name: 'about',
-    component: About,
-    meta: {
-      title: 'About',
-    },
-  },
-  {
-    path: '/about/maci',
-    name: 'about-maci',
-    component: AboutMaci,
-    meta: {
-      title: 'About MACI',
-    },
-  },
-  {
-    path: '/about/sybil-resistance',
-    name: 'about-sybil-resistance',
-    component: AboutSybilResistance,
-    meta: {
-      title: 'About Sybil Resistance',
-    },
-  },
-  {
-    path: '/about/layer-2',
-    name: 'about-layer-2',
-    component: AboutLayer2,
-    meta: {
-      title: 'About Layer 2',
-    },
-  },
-  {
-    path: '/about/how-it-works',
-    name: 'about-how-it-works',
-    component: AboutHowItWorks,
-    meta: {
-      title: 'How it works',
-    },
-  },
-  {
-    path: '/about/how-it-works/contributors',
-    name: 'about-how-it-works-contributors',
-    component: AboutContributors,
-    meta: {
-      title: 'Contributors Guide',
-    },
-  },
-  {
-    path: '/about/how-it-works/recipients',
-    name: 'about-how-it-works-recipients',
-    component: AboutRecipients,
-    meta: {
-      title: 'Recipients Guide',
-    },
-  },
-  {
-    path: '/about/public-goods',
-    name: 'about-public-goods',
-    component: AboutPublicGoods,
-    meta: {
-      title: 'About public goods',
-    },
-  },
-  {
-    path: '/about/quadratic-funding',
-    name: 'about-quadratic-funding',
-    component: AboutQuadraticFunding,
-    meta: {
-      title: 'About quadratic funding',
-    },
-  },
-  {
-    path: '/about/decentralization',
-    name: 'about-decentralization',
-    component: AboutDecentralization,
-    meta: {
-      title: 'About decentralization',
-    },
-  },
-  {
-    path: '/recipients',
-    name: 'recipients',
-    component: RecipientRegistryView,
-    meta: {
-      title: 'Recipient registry',
-    },
-  },
-  {
-    path: '/verify',
-    name: 'verify',
-    component: VerifyLanding,
-    meta: {
-      title: 'BrightID Verify Landing',
-    },
-  },
-  {
-    path: '/verify/success/:hash?',
-    name: 'verified',
-    component: Verified,
-    meta: {
-      title: 'Verified',
-    },
-  },
-  {
-    path: '/verify/:step',
-    name: 'verify-step',
-    component: VerifyView,
-    meta: {
-      title: 'Verification Steps',
-    },
-  },
-  {
-    path: '/join',
-    name: 'join',
-    component: JoinLanding,
-    meta: {
-      title: 'Recipient Join Form Landing',
-    },
-  },
-  {
-    path: '/join/success/:hash',
-    name: 'project-added',
-    component: ProjectAdded,
-    meta: {
-      title: 'Recipient Join Form Success',
-    },
-  },
-  {
-    path: '/join/:step',
-    name: 'join-step',
-    component: JoinView,
-    meta: {
-      title: 'Recipient Join Form Steps',
-    },
-  },
+    path: '/:factoryAddress(0x[a-fA-F0-9]{40})?',
```

Comment:
Should this wrap _every_ route? Seems kind of strange to e.g. have the about pages nested under the factory address: http://localhost:8080/#/0x0A8d125E155Ecb645668B7Ac00c7b6f196dF1AE1/about

Reply:
If it's easier I suppose I don't have a major issue.

Reply:
Yes, the reason I did it this way was because the About page is wrapped with round related components (like RoundInfromation) and also because of UX puroposes: as an user, if you are exploring an specific factory address, you wouldn't like to lose that address if you navigate to the /about page.

1. You are in `/:factoryAddress`
2. Then you go to `/about`
3. Then you go back to root `/` => you will be in a different factory